### DISCUSSION TITLE
Play a Random Album

### DISCUSSION THREAD

**Discussion by clriis** [rern/rAudio#1638] [Feature Requests]
@apastuszak way back in 2021 asked for a feature to play a random album from the library.

I really support that motion.

On my laptop I got this simple self-made script:

![Screenshot from 2024-03-25 10-30-33](https://github.com/rern/rAudio/assets/16999072/34dfd95b-d749-4bbe-9b68-a9b450ea708b)

I am not a code maker but am sure somebody would be able to make an extension to rAudio to make this possible.

Claus

**Comment by Editpo** [comment]
Not sure how long it’s been there but if you click the dice it gives you the option of random songs or random albums

**Comment by clriis** [comment]
@Editpo,

I feel a bit embarrased. I somehow always thought the dice was for manipulating the current playlist only.

Thank you very much.
